scout/parse/panel.py

```diff
@@ -196,63 +194,66 @@ def parse_genes(gene_lines):
         if not len(line) > 0:
             continue
         if line.startswith("#"):
-            if not line.startswith("##"):
-                # We need to try delimiters
-                # We prefer ';' or '\t' byt should accept ' '
-                line_length = 0
-                delimiter = None
-                for alt in delimiters:
-                    head_line = line.split(alt)
-                    if len(head_line) > line_length:
-                        line_length = len(head_line)
-                        delimiter = alt
-
-                header = [word.lower() for word in line[1:].split(delimiter)]
-        else:
-            # If no header symbol(#) assume first line is header
-            if i == 0:
-                line_length = 0
-                for alt in delimiters:
-                    head_line = line.split(alt)
-                    if len(head_line) > line_length:
-                        line_length = len(head_line)
-                        delimiter = alt
-
-                if "hgnc" in line or "HGNC" in line:
-                    header = [word.lower() for word in line.split(delimiter)]
+            if line.startswith("##"):
```

Comment:
Maybe this could be moved before `if line.startswith("#"):`?

Reply:
Fixed!